overlord/fdestate/backend/reseal.go

```diff
@@ -139,11 +189,22 @@ func resealRunObjectKeys(updateState StateUpdater, pbc boot.PredictableBootChain
 		return fmt.Errorf("at least one set of model-specific parameters is required")
 	}
 
+	if len(sigDBUpdate) > 0 {
+		logger.Debug("attaching DB update payload")
+		attachSignatureDBUPdate(modelParams, sigDBUpdate)
+	}
+
```

Comment:
Would it make sense to pull this up into resealKeys and drop the force reseal?

Reply:
no, reseal is done at 2 points, a pre-update step, where we pass the content which will be written to the EFI variable, and then a post-update step, where the reseal will effectively observe the content of the variable